src/serialize.jl

```diff
@@ -0,0 +1,18 @@
+const pickle = PyNULL()
+
+function get_pickle()
+    if pickle.o == C_NULL
+        copy!(pickle, pyimport("pickle"))
+    end
+
+    return pickle
+end
+
+function Base.serialize(s::AbstractSerializer, pyo::PyObject)
+    Base.serialize_type(s, PyObject)
+    serialize(s, get_pickle()[:dumps](pyo))
+end
+
+function Base.deserialize(s::AbstractSerializer, t::Type{PyObject})
+    get_pickle()[:loads](deserialize(s))
```

Comment:
After changing to use `PyBuffer` above, this should become:
```jl
b = deserialize(s) # Vector{UInt8}
pycall(get_pickle()["loads"], PyObject,
          PyObject(@pycheckn ccall(@pysym(PyString_FromStringAndSize),
                                        PyPtr, (Ptr{UInt8}, Int), b, sizeof(b))))
```
Note that you want to return a `PyObject` here, avoiding any conversion of the result to a Julia object.